src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/register_app_interface_request.h

```diff
@@ -127,13 +129,9 @@ class RegisterAppInterfaceRequest
    * @param app application with changed HMI status
    * @param resumption If true, resumption-related parameters will be sent to
    * the HMI
-   * @param need_restore_vr If resumption is true, whether or not VR commands
-   *should be resumed
    **/
   void SendOnAppRegisteredNotificationToHMI(
-      app_mngr::ApplicationConstSharedPtr app,
-      bool resumption = false,
-      bool need_restore_vr = false);
+      app_mngr::ApplicationConstSharedPtr app, bool resumption);
```

Comment:
I think that we don't need such enum, because now we even doesn't have separate `resumption` variable (here it's temporary variable, which is result of comparison `is_resumption_required && !is_resumption_failed_`). We don't use it in other places, so I think that it be better to leave it as is (local bool variable).